### PR TITLE
Add python3.13 build

### LINKED ISSUE #78: Is python 3.13 support coming?
### Comment:

Sorry for a somewhat lazy question, but is python 3.13 support coming?  Seems necessary for the build with python 3.13 here to work: conda-forge/postgresql-feedstock#209

Thank you

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
*  Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Ignorantly trying a python 3.13 build.
closes #78 
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by akrherz** [comment]
@conda-forge-admin please rerender

**Comment by akrherz** [comment]
@conda-forge-admin please rerender

**Comment by akrherz** [comment]
It is unclear to me why this worked on one OSX arch, but on none of the others.  Is repo metadata not fully updated yet? https://conda-metadata-app.streamlit.app/ seems to show it.  Anybody please feel free to push over what I have done here.

**Comment by jakirkham** [comment]
As this is still needed for Python 3.13, we've paused the migrator for now: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6361

cc @conda-forge/core

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
> It is unclear to me why this worked on one OSX arch, but on none of the others

Because we don't run tests on osx-arm, the runtime environment never gets resolved, so this didn't trigger.

File: recipe/conda_build_config.yaml
Comment (h-vetinari) [inline recipe/conda_build_config.yaml:1]: IMO we could drop all of
```yaml
version:
  - 3.8
  - 3.9
  - 3.7
  - 3.8
py_impl:
  - 73_pypy
  - 73_pypy
  - cpython
  - cpython
```
here, as we no longer build for those python versions.

**Pull request merged**